web/modules/custom/acceptance/joinup_acceptance.info.yml

```diff
@@ -1,8 +1,10 @@
 name: 'Joinup Acceptance'
-type: module
 description: 'Tasks to performed in the acceptance environment.'
-core_version_requirement: ^8 || ^9
+type: module
 package: Joinup
+core_version_requirement: ^8.9 || ^9
+php: 7.1
```

Comment:
I don't think we should add this. We should rely on Drupal core PHP minimum version. An exception could be the case when we need in a module version higher than the core min version. E.g. Drupal 8.9 works with PHP 7.1 but the module needs 7.3 in order to work. I guess we don't have such case.

Reply:
It is a QA coding standard rule that this PHP version has to be defined in the info file, so adding it is in scope of this PR. I agree that it is a bit stupid that we define the default value, but this is maybe something we should propose for the QA coding standards? For me personally it doesn't matter if this line is there or not.

We need minimum PHP 7.1 because we use nullable types and return types. We will for sure adopt 7.4 functionality as soon as we can :D